src/views/confirmation.njk

```diff
@@ -47,9 +47,9 @@
         <h2 class="govuk-heading-l">Reason for appeal</h2>
 
         {% set attachmentsList %}
```

Comment:
After setting nothing(?) to this variable is is then not used.
I wonder if the intention was 
`{% set attachmentsList = appealReasonDetails.attachments %}`
https://mozilla.github.io/nunjucks/templating.html#set

Reply:
**attachmentsList** is used as variable for the supporting document where the html content created inside it is just an ordered list if `appealReasonDetails.attachments` exists otherwise a simple `None string.
Used as an alternative for macros to have a clean njk page